Sources/Identity/CustomerInfoManager.swift

```diff
@@ -215,7 +205,12 @@ class CustomerInfoManager {
     func clearCustomerInfoCache(forAppUserID appUserID: String) {
         self.modifyData {
             $0.deviceCache.clearCustomerInfoCache(appUserID: appUserID)
-            $0.lastSentCustomerInfo = nil
```

Comment:
This didn't make sense. It didn't matter because the property wasn't used by anything, but now this is used for sending the "old" value. Resetting this broke an integration test, which is what I captured in the new `testCacheCustomerInfoSendsToDelegateAfterCachingComputedOnDevice` test.